### PR TITLE
Add shebang line to default template.py

### PR DESCRIPTION
**Pull request by Nodd**
It makes it easier to run scripts outside of spyder on linux.

The template could be further enhanced (add `if __name__ == "__main__"` for example).

### CODE REVIEW

File: spyderlib/plugins/editor.py [hunk 1]
```diff
@@ -382,8 +382,10 @@ def __init__(self, parent, ignore_last_opened_files=False):
         
         # Creating template if it doesn't already exist
         if not osp.isfile(self.TEMPLATE_PATH):
-            header = ['# -*- coding: utf-8 -*-', '"""', 'Created on %(date)s',
-                      '', '@author: %(username)s', '"""', '']
+            header = ['#!/usr/bin/env python' + ('2' if PY2 else '3'),
```
Comment (ccordoba12) [inline spyderlib/plugins/editor.py]: This line should be here only if we are not on Windows, i.e. `if not os.name == 'nt'`

**Comment by ccordoba12** [comment]
`if __name__ == "__main__"` is not for everybody and every situation, so I don't think we should include it.

**Comment by Nodd** [comment]
I agree about `if __name__ == "__main__"`. When #2884 is implemented it will be easily changed by the user.

**Comment by mdraw** [comment]
> The shebang shouldn't appear on windows now.

@Nodd Why would you leave out the shebang on Windows? The shebang line is also interpreted on Windows, see https://docs.python.org/3/using/windows.html#shebang-lines and https://www.python.org/dev/peps/pep-0397/.

**Comment by Nodd** [comment]
@mdraw I left it out because the boss (@ccordoba12) said so ! But since it's interpreted, I'll add it again.

**Comment by ccordoba12** [comment]
I'm -1 on adding a shebang to Windows scripts because:

1. It only works in Python 3.
2. It could confuse Windows users if it comes as `/usr/bin/env python`. I mean, it would (most probably) make them believe that that's an error or something. Experienced users (like @mdraw) can easily add the shebang themselves :-)

**Comment by Nodd** [comment]
I have some counter arguments, but since it's a detail and it's so easy to change I think it's not worth the time. I'll left it as is.

**Pull request merged**